src/authentication/create-account-method/index.tsx

```diff
@@ -24,9 +24,10 @@ export const CreateAccountMethod: React.FC<CreateAccountMethodProps> = ({ stage,
   ];
 
   const selectedOption = stage === RegistrationStage.WalletAccountCreation ? 'web3' : 'email';
+  const isWalletAccountCreationStage = stage === RegistrationStage.WalletAccountCreation;
```

Comment:
I'll lift this above line `selectedOption` and use `isWalletAccountCreateStage` like so:

`const selectedOption = isWalletAccountCreateStage ? 'web3' : 'email';`